docs/rules/no-nested-ternary.md

```diff
@@ -1,16 +1,16 @@
-# Disallow Nested Ternaries (no-nested-ternary)
+# disallow nested ternary expressions (no-nested-ternary)
 
-Nesting ternary expressions makes code unclear. The `no-nested-ternary` rule disallows the use of nested ternary expressions.
+Nesting ternary expressions can make code less readable. The `no-nested-ternary` rule disallows the use of nested ternary expressions.
```

Comment:
How about borrow from the previous rule “makes code more difficult to understand”

And delete the redundant second sentence.